src/accountsdb/genesis_config.zig

```diff
@@ -90,6 +109,17 @@ pub const Inflation = struct {
 
     /// DEPRECATED, this field is currently unused
     __unused: f64,
+
+    pub fn random(rand: std.Random) Inflation {
+        return .{
+            .initial = @floatFromInt(rand.int(u32)),
+            .terminal = @floatFromInt(rand.int(u32)),
+            .taper = @floatFromInt(rand.int(u32)),
+            .foundation = @floatFromInt(rand.int(u32)),
+            .foundation_term = @floatFromInt(rand.int(u32)),
+            .__unused = @floatFromInt(rand.int(u32)),
```

Comment:
Bitcasting would permit for the possibility of invalid or error-indicating representations such as NaN. But you raise a fair point, I'll look into some way to generate a uniformly distributed with better coverage.

Reply:
7ada3dc67e6218eea5110045cdd59840c499e390

Reply:
This covers much more of the floating point range, but it excludes negative numbers, and numbers greater than 2^32.

I think the only non-numeric bit patterns for floats are NaN, +Inf, -Inf, and indeterminate. Am I missing anything? The probability of encountering one of those is so small that it's reasonable to assume it would never happen in our lifetimes. For absolute certainty, they can be filtered out. Actually it may even be worth fuzzing with non-numbers to see what happens.

Reply:
Can't disagree, makes sense: 32c3aeb44729417bdae5487821186043efa20e43